environment.yml

```diff
@@ -17,11 +17,9 @@ dependencies:
 - beautifulsoup4=4.5*
 - pillow
 - lxml
-- networkx  # make sure nx installed even if no git
+- networkx=2.4*
```

Comment:
Is this equivalent to the `>=` statement in `requirements.txt`?

Reply:
No, this is ==. @rossbar would you mind updating to >=?

Reply:
Good question, according to the [conda environment docs](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) that's treated as a wildcard so the major and minor revision numbers must be 2.4 but the patch version number can be anything.

This isn't quite the same behavior as the requirements.txt which in principle would allow version 2.5, 2.6, etc. In that respect the `environment.yml` and the `requirements.txt` file are actually out-of-sync (`environment.yml` is lagging behind). I've documented this problem in a separate issue (#349)

Reply:
@jni I'm leaving it as-is in this PR because it turns out that the versions are pinned for most of the packages in the `environment.yml`. I will tackle that (discussed in further detail in #349) next.

It turns out I had missed a couple other `networkx` issues in `ch5.markdown` as well. I've made those changes as well. As a bonus: it looks like that's what was breaking the CI.